roofit/roofit/src/RooSDSCBShape.cxx

```diff
@@ -0,0 +1,246 @@
+/*****************************************************************************
+ * Project: RooFit                                                           *
+ * Package: RooFitModels                                                     *
+ * @(#)root/roofit:$Id$
+ * Authors:                                                                  *
+ *    T. Skwarnicki modify RooCBShape to Symmetrical Double-Sided CB         *
+ *    Michael Wilkinson add to RooFit source                                 *
+ *****************************************************************************/
+
+/** \class RooCBShape
+    \ingroup Roofit
+
+PDF implementing the Symmetrical Double-Sided Crystall Ball line shape.
```

Comment:
Latex formula if possible.

Reply:
done. I can create a plot after this is merged.